pages/documentation/getting-started-developers/phase-two.md

```diff
@@ -198,6 +198,7 @@ Setting | Default values - Version 2.x | Default values - Version 3.0 | Descript
 `sprite.projectIconsOnly` | `false` | `false` | Include _only_ the icons in `paths.src.projectIcons` in the icon sprite
 `settings.compile.browserslist` | `["> 2%", "last 2 versions", "IE 11", "not dead"]` | `["> 2%", "last 2 versions", "IE 11", "not dead"]` | Define the [browserslist query](https://github.com/browserslist/browserslist?tab=readme-ov-file#queries) for CSS prefixes generated by [autoprefixer](https://github.com/postcss/autoprefixer).
 `settings.compile.sassSourcemaps` | `true` | `true` | Include sourcemap when compiling SASS to CSS.
+`settings.compile.sassDeprecationWarnings` | `false` | `false` | Show USWDS Sass deprecation warnings. When set to `true`, Sass will output deprecation warnings for USWDS code in the terminal during compilation. Deprecation warnings for non-USWDS Sass will output even when this value is set to `false`.
```

Comment:
Looks good to me! One thing I did notice is we are using `<br>` tags to format the description column so the user isn't scrolling so much. If we want to keep that the same then I'd recommend adding in those `<br>` tags for formatting! Perhaps something like this:
```suggestion
`settings.compile.sassDeprecationWarnings` | `false` | `false` | Show USWDS Sass deprecation warnings. When set to `true`, Sass will output deprecation <br/>warnings for USWDS code in the terminal during compilation. Deprecation warnings for non-USWDS <br/>Sass will output even when this value is set to `false`.
```

Reply:
**praise:** Great catch here! Much better for the scrolling table